### DISCUSSION TITLE
Can axolotl be a desktop device to an android Signal app?

### DISCUSSION THREAD

**Discussion by bjmgeek** [axolotl-chat/axolotl#587] [Q&A] [answered]
I have an android phone that has Signal on it.  I can use the signal desktop app, but the desktop in question is a raspberry pi, so the official Signal desktop app is not supported.  Can I use axolotl in desktop mode?  I've installed axolotl via snapd, if that makes a difference.

**Comment by Flaburgan** [comment, marked as answer]
Hello, at the moment Axolotl is designed as a primary client for Signal, avoiding the need of an Android client. Unfortunately, you can have only one primary client so you can't use Axolotl to do what you want. We would like to add the possibility for Axolotl to act like a secondary client, this is track here: https://github.com/nanu-c/axolotl/issues/400 but this is low priority so not coming soon.
Otherwise, arm support for signal desktop was tracked here: https://github.com/signalapp/Signal-Desktop/issues/3410 but they closed that.
